scripts/install-nix-from-closure.sh

```diff
@@ -182,6 +182,15 @@ fi
 # shellcheck source=./nix-profile.sh.in
 . "$nix/etc/profile.d/nix.sh"
 
+NIX_LINK=$HOME/.nix-profile
```

Comment:
Why is it called "NIX_LINK", instead of something like NIX_PROFILE_DIR?
That's really a nitpick, but you've asked to read it through)

Reply:
Because it was named like that in some other place :)

I suppose this is a good opportunity to rename the var.